### DISCUSSION TITLE
Help Wanted

### DISCUSSION THREAD

**Discussion by grimsteel** [For-0/vocabustudy#156] [Announcements]
You need to know how to use `git`

**Comment by IMGROOT2** [comment]
Welp lol

**Reply by grimsteel** [reply]
Well...... you technically only need to know how to edit code with the GitHub UI but it's much easier if you can clone locally.
If you want I can add you as a collaborator to this repo

**Reply by grimsteel** [reply]
We should make a Vocabustudy Github Organization with @grimsteel @IMGROOT2 @DG3QRT1 @Nikhilatvocabustudy and maybe @ShreyanKhanna but he's not doing anything at all

**Reply by IMGROOT2** [reply]
Yeah! I want to learn a bit more about GitHub and Git to be safe, before I go around editing stuff lol

**Reply by grimsteel** [reply]
I will invite you anyway - I have protections so that you have to make a PR before editing anything (this was mainly for some of the... other ...collaborators on this repo

**Reply by IMGROOT2** [reply]
LOL thanks!

**Comment by grimsteel** [comment]
If I'm going to make an organization, we need a name
I feel like "Vocabustudy" is too specific because we might very well do other random stuff in the future

**Reply by IMGROOT2** [reply]
I have an interesting idea: "fourzero" or "forzero". Essentially the whole point of vocabustudy is to learn... for free, right? a.k.a learn for 0 dollars. and you can turn "for" into 4, which gives us "learn 4 0". "learn" can be replaced by whatever you want to do in the future, like "teach 4 0". so your "company" can be called "fourzero", and you can have different "applications", called "learn fourzero", or whatever.

it might be rlly dumb idk LOL

**Reply by grimsteel** [reply]
Genius!

**Reply by IMGROOT2** [reply]
Really? LOL

**Reply by grimsteel** [reply]
Fourzero is already a name of some random thing in Miami but Forzero seems viable

**Reply by grimsteel** [reply]
Ok I am doing "forzero"

It basically sums up the purpose of vocabustudy in two words lol

**Reply by IMGROOT2** [reply]
Oh nice that's great

**Reply by grimsteel** [reply]
oh both "forzero" and "fourzero" are already taken 😢

"forzero" is this random person who only has one repo that basically has nothing
![image](https://user-images.githubusercontent.com/115331356/197295557-6d126a62-0411-4ef5-ba82-f41b26d94ecb.png)

**Reply by IMGROOT2** [reply]
noooooooooo that's terrible

**Reply by IMGROOT2** [reply]
what about "four0" or "4zero"?

**Reply by grimsteel** [reply]
Both are taken...

Why are there so many people on github who have nothing at all in their accounts???

**Reply by grimsteel** [reply]
hmmm I can do "For 0" "4 Zero" or something else like "For NUL" (NUL is the windows stdout discard device I think)

**Reply by IMGROOT2** [reply]
It can't be too obscure like for nul, otherwise people won't catch the name

I'm at a loss right now. We might have to just think of something else.

**Reply by grimsteel** [reply]
Yes or we could do it with a space/dash like "four-0" or "4-zero"

**Comment by grimsteel** [comment]
If you want to help with Vocabustudy, but you don't know HTML/JS/CSS, sign up for the GitHub Student Developer pack!

There are many courses included that can teach you how to code

**Reply by grimsteel** [reply]
Thinkful seems like a good option - they say it teaches you html/css/js, git and GitHub!

**Comment by IMGROOT2** [comment]
hey, lol can I get write permissions to commit to a branch?

**Reply by IMGROOT2** [reply]
i'm pretty sure im set as triage idk

**Reply by grimsteel** [reply]
Done

**Reply by grimsteel** [reply]
Why are you committing at at like 1 AM?

**Reply by grimsteel** [reply]
We'll have to set a standard for 1 am changelogs rather than 11 pm ones

**Reply by IMGROOT2** [reply]
> Why are you committing at at like 1 AM?

oh no please don't ask LOL

**Reply by ShreyanKhanna** [reply]
LMAOOO

**Comment by grimsteel** [comment]
@Nikhilatvocabustudy @DG3QRT1  @ShreyanKhanna @For-0/vocabustudy-developers 

## Teams, Organizations, Discussions:
The For-0 Organization is everybody who could possibly make an application kind of like Vocabustudy - free for everyone. Most people in this org will get **Triage** access to the repo
The Vocabustudy Developers team (in the For-0 org) is JUST people who actively develop Vocabustudy. Anyone in this team gets **Write** access to the repo
For discussions that apply to the whole For 0 org or Vocabustudy, create it in the For 0 OR Vocabustudy discussions tab (they amount to the same thing)
For discussions that are about code or private discussions, create a PRIVATE discussion in the Vocabustudy Developers team

**Reply by grimsteel** [reply]
I know it seems like a lot but it provides structure

**Reply by ShreyanKhanna** [reply]
ok

**Reply by ShreyanKhanna** [reply]
also for getting accepted for the student thing in GitHub did you use ur school email?

**Reply by grimsteel** [reply]
> also for getting accepted for the student thing in GitHub did you use ur school email?

I used a home email, but I uploaded photos of my paper schedule and the one in HAC

**Reply by PrakmO** [reply]
ok
